index.html

```diff
@@ -661,24 +734,6 @@
                     mxEvent.consume(evt);
                 };
 
-                // Returns a shorter label if the cell is collapsed and no
-                // label for expanded groups
-                graph.getLabel = function(cell) {
```

Comment:
I remember that the removal of this function, affects something else. *Why* has this been removed?